src/main/java/SearchEngine.java

```diff
@@ -61,12 +63,17 @@ private void findPeoples(Map<String, List<Integer>> linesIndexesByWord) {
         }
     }
 
-
     private void printPeoples(List<String> peoples) {
         System.out.println("\n=== List of people ===");
         peoples.stream()
                 .distinct()
                 .map(String::trim)
                 .forEach(System.out::println);
     }
+
+    private void queryValidator(String[] queryWords) {
+        if (queryWords[0].equals("")) {
```

Comment:
Can you use the `empty` method instead of the `equals` one?

Reply:
used the empty array check